ts-client/src/ValorClient.ts

```diff
@@ -0,0 +1,350 @@
+import axios, { AxiosInstance } from 'axios';
+
+type Dataset = {
```

Comment:
These types might need to be exported so that I can import them into Horus.

Reply:
done